addons/l10n_nl/migrations/3.2/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,18 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    nl_country_id = env['res.country'].search([('code', '=', 'NL')], limit=1).id
+    tags_3b = env['account.account.tag']._get_tax_tags('3b (omzet)', nl_country_id)
+    tags_3bl = env['account.account.tag']._get_tax_tags('3bl (omzet)', nl_country_id)
+    tag_3b_plus = tags_3b.filtered(lambda t: t.name == '+3b (omzet)')
+    tag_3b_minus = tags_3b.filtered(lambda t: t.name == '-3b (omzet)')
+    tag_3bl_plus = tags_3bl.filtered(lambda t: t.name == '+3bl (omzet)')
+    tag_3bl_minus = tags_3bl.filtered(lambda t: t.name == '-3bl (omzet)')
+    lines = env['account.move.line'].search([('tax_tag_ids', 'in', tags_3b.ids), ('tax_tag_ids', 'not in', tags_3bl.ids)])
```

Comment:
You'd better do some SQL than that. On big databases, I don't think this will work well.